src/main/webapp/app/admin/statistics/statistics.component.ts

```diff
@@ -0,0 +1,165 @@
+import { Component, OnInit, ViewChild } from '@angular/core';
+import { StatisticsService } from 'app/admin/statistics/statistics.service';
+import { ChartDataSets, ChartOptions, ChartType } from 'chart.js';
+import { BaseChartDirective, Label } from 'ng2-charts';
+import { DataSet } from 'app/exercises/quiz/manage/statistics/quiz-statistic/quiz-statistic.component';
+import { TranslateService } from '@ngx-translate/core';
+import * as moment from 'moment';
+import { SpanType } from 'app/entities/statistics.model';
+
+@Component({
+    selector: 'jhi-statistics',
+    templateUrl: './statistics.component.html',
+})
+export class StatisticsComponent implements OnInit {
+    DAY = SpanType.DAY;
+    WEEK = SpanType.WEEK;
+    MONTH = SpanType.MONTH;
+    YEAR = SpanType.YEAR;
+    span: SpanType = SpanType.WEEK;
```

Comment:
Is this the currently active span? If so it's better to rename this.

Reply:
Thanks, adjusted it in [this](https://github.com/ls1intum/Artemis/pull/2471/commits/5905879e8731f464bb3777cae0b6e686a48611d8) commit